### PR TITLE
Adds MultiClusterFetcher instructions to user-guide

### PR DESCRIPTION
**Pull request by foot**
Updates documentation around how to configure MultiClusterFetcher things

### CODE REVIEW

File: website/docs/cluster-management/generic-clusters.mdx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+---
+
+import TierLabel from "../_components/TierLabel";
+
+# Generic Cluster <TierLabel tiers="enterprise" />
+
+### Managing non-capi clusters
+
+Any kubernetes cluster whether capi or not can be added to Weave Gitops Enterprise.
+
```
Comment (foot) [inline website/docs/cluster-management/generic-clusters.mdx:14]: Describe a scenario where you have just created a terraform k8s cluster.
- If you have a kubeconfig as a result of this process describe how to add it to a secret and then create a `GitopsCluster`.
- If you don't have a kubeconfig or a short lived one then get ppl to follow the below instructions. Then add a `GitopsCluster` object.

File: website/docs/cluster-management/generic-clusters.mdx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+- TOKEN: the token of the service account retrieved in the previous step
+
+Finally create a secret for the generated kubeconfig:
+
+```sh
+kubectl create secret generic demo-01-kubeconfig \
+--from-file=value.yaml=./demo-01-kubeconfig \
+--dry-run=client -o yaml > demo-01-kubeconfig.yaml
+```
+
```
Comment (foot) [inline website/docs/cluster-management/generic-clusters.mdx:107]: HERE: Describe how to create a `GitopsCluster` object w/. secretRef and to commit to `./clusters/management/clusters/my-cluster.yaml`

File: website/docs/cluster-management/generic-clusters.mdx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+4. Create a kubeconfig secret
+
+For the next step, the cluster certificate (CA) is needed. How you get hold of the certificate depends on the cluster. For GKE you can view it on the GCP Console: Cluster->Details->Endpoint->”Show cluster certificate”. You will need to copy the contents of the certificate into the `ca.crt` file used below.
+
+```sh
+CLUSTER_NAME=demo-01 \
+CA_CERTIFICATE=ca.crt \
+ENDPOINT=<control-plane-ip-address> \
+TOKEN=<token> ./tools/sa/static-kubeconfig.sh > demo-01-kubeconfig
```
Comment (foot) [inline website/docs/cluster-management/generic-clusters.mdx:90]: include this script as an appendix to this section

File: website/docs/cluster-management/assets/rbac/wego-admin.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    verbs: ["get", "list", "patch"]
+  - apiGroups: ["helm.toolkit.fluxcd.io"]
+    resources: ["helmreleases"]
+    verbs: ["get", "list", "patch"]
+  - apiGroups: ["source.toolkit.fluxcd.io"]
+    resources: ["buckets", "helmcharts", "gitrepositories", "helmrepositories"]
+    verbs: ["get", "list", "patch"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["get", "watch", "list"]
```
Comment (foot) [inline website/docs/cluster-management/assets/rbac/wego-admin.yaml:37]: make sure this is up to date, and add policy permissions

**Review by saeedfazal — APPROVED** [review]
:100:

**Comment by foot** [comment]
Alright maybe we clean this up so that the "Getting started" guide branches out into CAPI or Connect-a-cluster

**Pull request merged**